### DISCUSSION TITLE
Connecting 2 cameras over WiFi

### DISCUSSION THREAD

**Discussion by ibiz01** [gopro/OpenGoPro#298] [Q&A]
Hello,

my setup: OS Windows, 2 USB Wi-Fi adapters and 2 GoPros 10.
I run 2 (identical) python scripts, each of them connects to a specific camera through a specific Wi-Fi adapter.

The piece of code for connecting:

`with WirelessGoPro(wifi_interface="Wifi 2", target="GoPro 1273") as gopro:
`

As long as I run only 1 script, everything works well.

When I run both scripts, both cameras get ble connected, but, each time an http command is sent (e.g. shutter command), it's received only by one camera, whilst the other camera does not receive it. I guess the problem might depend on both cameras having the same IP address? 
(When I use ble commands for the shutter, it works smooth, but unfortunately ble connection does not provide media management which I need)

Is there something wrong in my code?
In general, is it possible to Wi-Fi connect to multiple cameras?

Many thanks

**Comment by jonericcook** [comment]
@ibiz01 check out my question here https://github.com/gopro/OpenGoPro/discussions/71
